web-components/src/[sandbox]/examples/form.ts

```diff
@@ -0,0 +1,18 @@
+import "@/components/form/Form";
+import { html } from "lit-element";
+
+export const formTemplate = html`
+  <h3>Default</h3>
+  <md-form is-valid>
+    <md-input></md-input>
+    <md-button type="submit">Submit</md-button>
```

Comment:
Why have this button here? It doesn't do anything in sandbox.

Also, for this default form, should the entire page refresh when the user invokes the enter key on the input field?
Also, maybe there should be a custom event that fires? or is it the Input web component that will handle that?

Another thing, we should include an example in sandbox for a form with label/input pairing and confirm that the result is in a format similar to `name=Hi&lname=Do `.

Reply:
@elenagiere thx for good questions. 

> should the entire page refresh when the user invokes the enter key on the input field?

This is native behaviour of any form that contain [submittable](https://developer.mozilla.org/en-US/docs/Learn/Forms/Form_validation#validating_forms_using_javascript) element. 

> Also, maybe there should be a custom event that fires?

For this purpose, my point that native event bubbling will be enough.

> Another thing, we should include an example in sandbox for a form with label/input pairing and confirm that the result is in a format similar to name=Hi&lname=Do 

This will be useful - will do